### PR TITLE
Feat: Update related_website_sets.JSON, add CTV sites

### PR DESCRIPTION
**Pull request by gersveo**

### CODE REVIEW

**Review by shuranhuang — APPROVED** [review]
LGTM, merging.

**Review by Parasos — COMMENTED** [review]
related_website_sets.JSON

**Comment by Parasos** [comment]
related_website_sets.JSON

File: related_website_sets.JSON [hunk 1]
```diff
@@ -571,6 +571,24 @@
     		"https://startupislandtaiwan.net": "Domain alias", 
     		"https://startupislandtaiwan.org": "Domain alias"
     	}
+    },
+    {
+      "contact": "addigital@caracoltv.com.co",
+      "primary": "https://caracoltv.com",
```
Comment (Parasos) [inline related_website_sets.JSON:577]: related_website_sets.JSONRWS-Submission_Guidelines.mdRwsCheck.py

**Review by Parasos — COMMENTED** [review]
related_website_sets.JSON

**Comment by Parasos** [comment]
gh

**Pull request merged**